### DISCUSSION TITLE
Fly.io How To Integrate Google Search Console.

### DISCUSSION THREAD

**Discussion by JohnMatychuk** [towfiqi/serpbear#140] [Q&A] [answered]
I've generated the api keys but I'm not sure how to integrate with the instance of Serpbear I have running on Fly.io.
Any idea how to get this up and running? Thanks!

**Comment by towfiqi** [comment, marked as answer]
Assuming you have already deployed to fly.io using this doc: https://docs.serpbear.com/deployment/deploying-to-fly.io-free
All you have to do is run this command from your terminal:

`fly secrets set --app YOUR__FLY.IO_APP_NAME SEARCH_CONSOLE_PRIVATE_KEY=SEARCH_CONSOLE_PRIVATE_KEY=-----BEGIN PRIVATE KEY-----\ssssaswdkihadho\nlnb/Kt88qQC...KYv6A==\n-----END PRIVATE KEY-----\n`

And then run:

`fly secrets set --app YOUR__FLY.IO_APP_NAME SEARCH_CONSOLE_CLIENT_EMAIL=myemail@appspot.gserviceaccount.com`

Replace the `YOUR__FLY.IO_APP_NAME` before running the commands.

**Reply by JohnMatychuk** [reply]
Thanks for your help! I wasn't able to get this to work via the terminal but I was able to set the secrets via the Fly.io dashboard.
Thanks for building this great tool!

**Comment by XahidEx** [comment]
I'm facing an issue. Fly.io is like automatically erases everything. I've added everything (domain+api+keywords) after 5-10 minutes, it gets automatically erased. It's happening again and again. How to solve this, Does anyone know?

**Reply by towfiqi** [reply]
It's because you have not installed serpbear properly on your fly.io instance. Since fly.io instances storage is not persistent, you will have to create a volume, and then assign that volume to your fly.io serpbear app instance. Make sure you have followed the instructions posted here: https://docs.serpbear.com/deployment/deploying-to-fly.io-free

**Reply by XahidEx** [reply]
SOLVED :)
